src/components/ui/WithTooltip/WithTooltip.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { WithTooltip } from '.'
+
+<Meta title="UI Components/WithTooltip" />
+
+# WithTooltip
+
+Returns an HOC that renders a tooltip. Make sure to pass an existing element container ID and spread props of the wrapped component.
```

Comment:
stupid question. What is a HOC?

Reply:
It's a generally used React term -- essentially a function that takes in a component and returns a new wrapped component with additional functionality

[From the docs](https://reactjs.org/docs/higher-order-components.html#:~:text=A%20higher%2Dorder%20component%20(HOC,React%20for%20reusing%20component%20logic.&text=They%20are%20a%20pattern%20that,and%20returns%20a%20new%20component )

Reply:
To clarify, `HOC` is s term commonly used in functional programming, not just React.

Reply:
I think you mean `HOF` (higher order function). But yeah, HOC isn't specific to React but it is a fairly common term for those working with the framework